site-footer.md

```diff
@@ -0,0 +1,20 @@
+### Help & Support
+
+ Project Coordinators
+ 
+ [Beatriz Serrano-Solano](#)
+ 
+ [Berenice Batut](#)
+ 
+ [Assunta DeSanto](#)
+ 
+ ### Contact Us
+ 
+  Email: galaxy.mentoring@gmail.com
```

Comment:
```suggestion
- Email: [galaxy.mentoring@gmail.com](mailto:galaxy.mentoring@gmail.com)
```

Reply:
I didn't know how to add the email icon there, I figured I'll add it in the HTML editor. Is there a way to do it in markdown, please?